src/components/LoginForm/LoginForm.vue

```diff
@@ -3,6 +3,7 @@ import UiTextInput from '@/components/common/UiInput/UiInput.vue'
 import UiButton from '@/components/common/UiButton/UiButton.vue'
 import googleIcon from '@/assets/icons/google-icon.svg'
 import { useI18n } from 'vue-i18n'
+import UiImage from '../common/UiImage/UiImage.vue'
```

Comment:
```suggestion
import UiImage from '@/components/common/UiImage/UiImage.vue'
```
Так потом рефакторить будет легче)